src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="www"
+      method="get"
+      data-qa="big"
+      class="searchform searchformbig"
+    >
+      <label class="field">
+        <input
+        type="search"
+        class="field bigsearch"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+        <button
+          type="submit"
+          class="button buttonbig">
+        </button>
```

Comment:
Go through [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) one more time and fix your code style, attributes shold have indents, same with closing brackets